x/dex/keeper/core.go

```diff
@@ -105,19 +101,7 @@ func (k Keeper) DepositCore(
 		}
 
 		if inAmount0.Equal(sdk.ZeroInt()) && inAmount1.Equal(sdk.ZeroInt()) {
-			ctx.EventManager().EmitEvent(types.CreateDepositFailedEvent(
```

Comment:
Why are we removing this?

Reply:
(from discord)

its not needed since the whole tx will be failed - none of the other tx failures we have result in an event being logged
it was there previously because the tx would continue on partial failure and the events were needed by the front end to piece together what actually occured